### ISSUE TITLE
[Bug]: Magazine Not Change

### ISSUE THREAD

**Issue by Rip0000** [MCModderAnchor/TACZ#276]
### I have confirmed that ...

- [X] tacz is up to date
- [X] unable to reproduce without tacz

### Minecraft and modloader version

1.20.1 / Forge 47.3.7

### tac version

tacz-1.20.1-1.0.3-all

### Related Mods

```raw
I'm playing Atm9 Mod pack version 0.3.2
```

### Description

Hello, I'm playing Atm9 added it to the package because I liked the TacZ mod. I can produce the weapons added by the mod without any problems. I also produce their bullets, but when the bullet in the magazine runs out, the magazine does not change.
btw sorry for my bad english.
This is my first time opening a ticket, I apologize for my ignorance.

**Comment by xjqsh** [comment]
don't really understand what `the magazine does not change` means, can you explain what you are trying to say? did u mean failed to reload or something else?

**Comment by Rip0000** [comment]
yes not reload Even the animation doesn't work and doesn't change even when the bullet runs out, even in creative mode.
